daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala

```diff
@@ -1856,7 +1858,10 @@ object SBuiltinTest {
       sexpr: SExpr,
       getContract: PartialFunction[Value.ContractId, Value.VersionedContractInstance],
       onLedger: Boolean,
-  ): Either[SError, (SValue, Map[ContractId, CachedContract])] = {
+  ): Either[
```

Comment:
Note for another clean up PR. 

This looks very similar to code from CompilerTest. 
This should probably be factoring in `SpeedyTestLib`,